client-interfaces/src/main/java/org/opentestsystem/shared/progman/init/ProgManRetriever.java

```diff
@@ -39,7 +39,7 @@
     @Autowired(required=false)
     private ProgManClient progManClient;
 
-    @Value("#{systemProperties['progman.locator']}")
+    @Value("${progman.locator}")
```

Comment:
This is the big change.  I'm not sure why it was originally restricted to ONLY resolving a system property...